kernel/src/eval/evaluator.cpp

```diff
@@ -28,6 +28,22 @@
 
 ////////////////////////////////////////////////////////////////////////////////
 
+
+#if defined(__GNUC__) && !defined(__clang__) && __GNUC__ < 5
+// http://stackoverflow.com/questions/27064791/stdalign-not-supported-by-g4-9
+namespace std {
+    inline void *align( std::size_t alignment, std::size_t size, void *&ptr, std::size_t &space ) {
```

Comment:
Hey @sethalves, you probably want to use the corrected version provided at the end of the thread [here](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=57350) to avoid some alignment issues.

I believe the corrected function is
```
inline void *align( std::size_t alignment, std::size_t size,
                    void *&ptr, std::size_t &space ) {
	std::uintptr_t pn = reinterpret_cast< std::uintptr_t >( ptr );
	std::uintptr_t aligned = ( pn + alignment - 1 ) & - alignment;
	std::size_t padding = aligned - pn;
	if ( space < size + padding ) return nullptr;
	space -= padding;
	return ptr = reinterpret_cast< void * >( aligned );
}
```
This is also the version that's made it into `gcc` trunk ([here](https://gcc.gnu.org/viewcvs/gcc/trunk/libstdc%2B%2B-v3/include/std/memory?view=markup&pathrev=216149)) beyond 4.9 if I understand correctly.